common/core/src/test/java/com/vmware/connectors/common/CardTest.java

```diff
@@ -29,10 +29,10 @@
 
 import static java.util.concurrent.TimeUnit.MILLISECONDS;
 import static org.hamcrest.CoreMatchers.*;
-import static org.junit.Assert.assertThat;
+import static org.hamcrest.MatcherAssert.assertThat;
 
 
-public class CardTests {
+public class CardTest {
 
```

Comment:
Renamed so that default settings of `Maven Surefire Plugin` will find these classes.